website/docs/r/ruleset_rule.html.markdown

```diff
@@ -23,34 +23,48 @@ resource "pagerduty_ruleset" "foo" {
     id = pagerduty_team.foo.id
   }
 }
+
+# The pagerduty_ruleset_rule.foo rule defined below
+# repeats daily from 9:30am - 11:30am using the America/New_York timezone.
+# Thus it requires a time_static instance to represent 9:30am on an arbitrary date in that timezone.
+# April 11th, 2019 was EDT (UTC-4) https://www.timeanddate.com/worldclock/converter.html?iso=20190411T133000&p1=179
+resource "time_static" "eastern_time_at_0930" {
+  rfc3339 = "2019-04-11T09:30:00-04:00"
+}
+
 resource "pagerduty_ruleset_rule" "foo" {
   ruleset = pagerduty_ruleset.foo.id
   position = 0
   disabled = "false"
   time_frame {
     scheduled_weekly {
-	  weekdays = [3,7]
-	  timezone = "America/Los_Angeles"
-	  start_time = "1000000"
-	  duration = "3600000"
-	}
+      # Every Tuesday, Thursday, & Saturday
+      weekdays = [2, 4, 6]
+      # Starting at 9:30am
+      start_time = "${time_static.eastern_time_at_0930.unix * 1000}"
```

Comment:
We're trying to standardize on the syntax introduced in Terraform v0.12 (i think). Please change this to match that notation as well. Essentially, just removing the double-quotes, dollar sign and curly-brackets. :) Something like this

`start_time = time_static.eastern_time_at_0930.unix * 1000`